target/linux/generic/hack-5.10/780-3-net-dsa-implement_ndo_set_netlink_for_chaning_ports_CPU_port.patch

```diff
@@ -0,0 +1,152 @@
+From mboxrd@z Thu Jan  1 00:00:00 1970
+Return-Path: <SRS0=lHAO=WU=vger.kernel.org=netdev-owner@kernel.org>
+X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
+	aws-us-west-2-korg-lkml-1.web.codeaurora.org
+X-Spam-Level: 
+X-Spam-Status: No, score=-9.9 required=3.0 tests=DKIM_SIGNED,DKIM_VALID,
+	DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_PATCH,MAILING_LIST_MULTI,
+	SIGNED_OFF_BY,SPF_HELO_NONE,SPF_PASS,USER_AGENT_GIT autolearn=ham
+	autolearn_force=no version=3.4.0
+Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
+	by smtp.lore.kernel.org (Postfix) with ESMTP id B6EF9C3A5A2
+	for <netdev@archiver.kernel.org>; Sat, 24 Aug 2019 02:43:09 +0000 (UTC)
+Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
+	by mail.kernel.org (Postfix) with ESMTP id 8DC852173B
+	for <netdev@archiver.kernel.org>; Sat, 24 Aug 2019 02:43:09 +0000 (UTC)
+Authentication-Results: mail.kernel.org;
+	dkim=pass (1024-bit key) header.d=nic.cz header.i=@nic.cz header.b="LOMQONxV"
+Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
+        id S1726936AbfHXCnH (ORCPT <rfc822;netdev@archiver.kernel.org>);
+        Fri, 23 Aug 2019 22:43:07 -0400
+Received: from mail.nic.cz ([217.31.204.67]:37288 "EHLO mail.nic.cz"
+        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
+        id S1725924AbfHXCnC (ORCPT <rfc822;netdev@vger.kernel.org>);
+        Fri, 23 Aug 2019 22:43:02 -0400
+Received: from dellmb.labs.office.nic.cz (unknown [IPv6:2001:1488:fffe:6:cac7:3539:7f1f:463])
+        by mail.nic.cz (Postfix) with ESMTP id D9A45140D23;
+        Sat, 24 Aug 2019 04:42:59 +0200 (CEST)
+DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=nic.cz; s=default;
+        t=1566614580; bh=AC+PVQwA6Fix9qcvNMGUHHTWu5CWtzFdGlYjHQ3j5AQ=;
+        h=From:To:Date;
+        b=LOMQONxVq2PFB4CLqpcuBlTBtVaCwpHmEGDYOjWDWTq0PhBYF/GXGsjK8VHv+kbjj
+         fQ9bsLsqog0Ln0Zi9M04t4krFY8xbQySLesWaQGbTjc2coKXUwCvsRH8Z70TLUuz8C
+         sgnXbNcMR6nWd8S/InLWwiTOlkMeAz3jX0ZIw8NE=
+From:   =?UTF-8?q?Marek=20Beh=C3=BAn?= <marek.behun@nic.cz>
+To:     netdev@vger.kernel.org
+Cc:     Andrew Lunn <andrew@lunn.ch>,
+        Vivien Didelot <vivien.didelot@gmail.com>,
+        Florian Fainelli <f.fainelli@gmail.com>,
+        David Ahern <dsahern@gmail.com>,
+        Stephen Hemminger <stephen@networkplumber.org>,
+        =?UTF-8?q?Marek=20Beh=C3=BAn?= <marek.behun@nic.cz>
+Subject: [PATCH RFC net-next 3/3] net: dsa: implement ndo_set_netlink for chaning port's CPU port
```

Comment:
You might fix the typo in the title (and patch file name)
780-3-net-dsa-implement_ndo_set_netlink_for_chaning_ports_CPU_port.patch 

chaning ???  chaining or changing ?

That seems to come from the upstream, but it might still be fixed (at least our own file name)

Reply:
changing  should be the fix.